tools/server/src/wsgi.py

```diff
@@ -0,0 +1,14 @@
+"""
```

Comment:
Thats actually from the django-admin generator which I'd like to keep as standard as possible (so its easier for upgrade)